src/www/messages/en.json

```diff
@@ -14,6 +14,9 @@
   "disconnected-server-state": "Disconnected",
   "disconnecting-server-state": "Disconnecting...",
   "email-feedback-input": "Email address (optional)",
+  "error-connection-configuration": "The configuration information for your server is incomplete or invalid. Please contact your service manager to troubleshoot this issue.",
+  "error-connection-configuration-fetch": "Outline failed to download the configuration information for your server. Please contact your service manager to troubleshoot this issue.",
```

Comment:
I like the direction to "Please contact your service manager"!

Reply:
😄 Customers might think, "Ah, service manager = Outline"

Reply:
Doubt anyone will know who the "service manager" is. maybe "...the administrator who gave you this access key" since "contact your administrator" is sometimes used this way in other applications. Or even more plainly "...the person who gave you this access key"

Reply:
Reasonable point. We do say "service provider" in our help documentation but that doesn't mean it's clear. We also need to handle the case of people who got theirs from a bot or a website or some other non-personal contact.

Reply:
I definitely support "the person who gave you this access key"-- simple, people will know what that means without us having to define what an "administrator" or "service manager" is.

As a bonus, it uses simpler wording that's easier to translate into other languages.